lib/src/buttons/pulldown_button.dart

```diff
@@ -26,6 +26,7 @@ enum PulldownMenuAlignment {
 
 // The widget that is the button wrapping the menu items.
 class _MacosPulldownMenuItemButton extends StatefulWidget {
+  // ignore: use_super_parameters
```

Comment:
I also see that changing this to super.key, makes the analyzer complain about unused variable. Why could that be? It's the same code as the MacosPopupMenuItemButton

Reply:
Yeah I have no idea what's up with  that. As far as I could tell there's no reason for the analyzer to complain, maybe it's a bug in the linter?

Reply:
Yes, I expect it will be fixed in the future.